jpa-persistence-lib-demo/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.axonivy.market</groupId>
+  <artifactId>jpa-persistence-lib-demo</artifactId>
+  <version>9.3.0-SNAPSHOT</version>
+  <packaging>iar</packaging>
+  <properties>
+    <axonivy.persistence.version>0.10.0</axonivy.persistence.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>com.axonivy.utils</groupId>
+      <artifactId>persistence-utils</artifactId>
+      <version>${axonivy.persistence.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-jpamodelgen</artifactId>
+      <version>5.4.21.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate.javax.persistence</groupId>
+      <artifactId>hibernate-jpa-2.1-api</artifactId>
+      <version>1.0.0.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>org.quartz-scheduler</groupId>
+      <artifactId>quartz</artifactId>
+      <version>2.1.1</version>
+    </dependency>
```

Comment:
These dependencies can be removed.

Reply:
hibernate-jpamodelgen is required for the meta model generation.

Reply:
I do not get any runtime or compile errors without hibernate-jpamodelgen. Where does this dependency come into paly?